lib/simulation/openfisca/mapping/index.js

```diff
@@ -203,17 +210,27 @@ var mapFamille = function(situation) {
 var mapFoyerFiscal = exports.mapFoyerFiscal = function(situation) {
     var foyerFiscal = transform(situation, foy, situation);
     var ressources = [];
-    _.forEach(situation.individus, function(individu) { ressources = ressources.concat(individu.ressources); });
+    _.forEach(situation.individus, function(individu) {
+        ressources = ressources.concat(individu.ressources);
+    });
     applyRessources({ ressources: ressources }, foyerFiscal, ressourcesMap.foyerFiscal, situation.dateDeValeur);
 
     return foyerFiscal;
 };
 
-exports.mapSituation = function(situation) {
-    return {
-        familles: [mapFamille(situation)],
-        foyers_fiscaux: [mapFoyerFiscal(situation)],
-        individus: mapIndividus(situation),
-        menages: [transform(situation, men, situation)]
+exports.buildOpenFiscaRequest = function(situation, debug) {
+        return {
+        intermediate_variables: debug,
```

Comment:
It seems pretty annoying to pass this `debug` param over and over. Might be worth looking into whether we can replace it with `process.env.DEBUG` or `process.env.NODE_ENV == 'dev'` someday.